pkg/interop/native/std/std.go

```diff
@@ -146,3 +146,18 @@ func MemorySearchFull(mem, pattern []byte, start int, backward bool) int {
 	return contract.Call(interop.Hash160(Hash), "memorySearch", contract.NoneFlag,
 		mem, pattern, start, backward).(int)
 }
+
+// StringSplit splits s by ocurrences of sep.
+// It uses `stringSplit` method of StdLib native contract.
+func StringSplit(s, sep string) []string {
+	return contract.Call(interop.Hash160(Hash), "stringSplit", contract.NoneFlag,
+		s, sep).([]string)
+}
+
+// StringSplitNonEmpty splits s by ocurrences of sep.
+// If removeEmpty is true, remove empty items from the resulting slice.
+// It uses `stringSplit` method of StdLib native contract.
+func StringSplitNonEmpty(s, sep string, removeEmpty bool) []string {
```

Comment:
Maybe we can drop `removeEmpty` parameter here? `StringSplit` will do what it does and `StringSplitNonEmpty` will just call add `true` to parameter list?

Reply:
I agree that the interface will be cleaner. Here I have followed the policy "of 1-1 wrappers".